src/main/java/org/cryptomator/common/locationpresets/DropboxLinuxLocationPresetsProvider.java

```diff
@@ -0,0 +1,35 @@
+package org.cryptomator.common.locationpresets;
+
+import org.cryptomator.integrations.common.OperatingSystem;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import static org.cryptomator.integrations.common.OperatingSystem.Value.LINUX;
+
+@OperatingSystem(LINUX)
+public final class DropboxLinuxLocationPresetsProvider implements LocationPresetsProvider {
+
+	private static final Path USER_HOME = LocationPresetsProvider.resolveLocation("~/.").toAbsolutePath();
+	private static final Pattern PATTERN = Pattern.compile("Dropbox \\(.+\\)");
+
+	@Override
+	public Stream<LocationPreset> getLocations() {
+		try (var dirStream = Files.newDirectoryStream(USER_HOME,"Dropbox*")){
+			return StreamSupport.stream(dirStream.spliterator(), false).flatMap(p -> {
```

Comment:
This is already a stream, you can map directly on it. Not sure, how to properly close it though, better check the docs!

Reply:
No, unfortunately, a [`DirectoryStream`](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/nio/file/DirectoryStream.html) does not implement the Stream interface.

Reply:
[`Files.list` does, though](https://stackoverflow.com/q/33668859/4014509)

Reply:
But does not offer a glob filter, which is implemented as a filesystem specific path matcher😉  And we would still need to wrap everything in a try-with-resource block catching an UncheckedIO exception.

Reply:
But we're filtering using `PATTERN` anyway (the glob doesn't do anything else).

Furthermore, please note that we're `return`ing the Stream here. So we *must not* close the dir handle before consuming the stream! We must make sure to close the Stream then.

Reply:
Good point. I circumvented that problem by collectiong the entries into a list and rather stream the list. Fixed in https://github.com/cryptomator/cryptomator/pull/2918/commits/b1a3ef9023de69131e4e9bbe1444a30cf72a8973